src/app/clusters/bindings/bindings.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/util/binding-table.h>
+
+namespace chip {
+
+constexpr char kBindingStoargeKey[] = "binding";
```

Comment:
please add some documentation: where is this storage key relevant?

Below, what is a binding entry and where is it added? Explain what globals or storage this is regarding with. When you add a binding it, how do you fetch it? Please give some examples and usage rationale.